cmd/livepeer/starter/starter.go

```diff
@@ -1412,3 +1417,31 @@ func getBroadcasterPrices(broadcasterPrices string) []BroadcasterPrice {
 
 	return pricesSet.Prices
 }
+
+type KeystorePath struct {
+	path, address interface{}
+}
+
+func ParseEthKeystorePath(ethKeystorePath string) (KeystorePath, error) {
+	var keystore KeystorePath = KeystorePath{"", ""}
+	ethKeystorePath = strings.TrimSuffix(ethKeystorePath, "/")
+
+	if fileInfo, err := os.Stat(ethKeystorePath); !os.IsNotExist(err) && fileInfo != nil {
+		if fileInfo.IsDir() {
+			keystore.path = ethKeystorePath
+		} else {
+			keystore.path, _ = filepath.Split(ethKeystorePath)
+			if keyText, err := common.ReadFromFile(ethKeystorePath); err == nil {
```

Comment:
I think that if `err != nil`, then we should just return an error and fail-fast. Do we want to proceed in case of a wrong path?

Reply:
Updated logic flow to return "error opening keystore" when path or file not found